src/3rdparty/walkontable/test/spec/table.spec.js

```diff
@@ -203,10 +203,10 @@ describe('WalkontableTable', () => {
     // expectWtTable(wt, wtTable => wtTable.getCell({row: -1, col: -1}), 'bottom').toBe(undefined); // TODO this should be -1 or not a callable method, but it throws
```

Comment:
Is it still a TODO thing?

Reply:
I think not, I will remove this line.

However, the line 202 is still a TODO.